index.html

```diff
@@ -0,0 +1,220 @@
+                </div>
+            </div>
+            <div class="grid-item">
+                <div class="grid-item-top"></div>
+                <div class="grid-item-bottom">
+                    <h4 class="header-1">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="image-container">
+                        <li><a href=""><img src="img/ruby.png" alt="ruby"></a></li>
+                        <li><a href=""><img src="img/css.png" alt="css"></a></li>
+                        <li><a href=""><img src="img/javascript.png" alt="javascript"></a></li>
+                        <li><a href=""><img src="img/html.png" alt="html"></a></li>
+                    </ul>
+                    <button class="btn" type="button"><span class="text">See project</span></button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <div class="grid-item-top"></div>
+                <div class="grid-item-bottom">
+                    <h4 class="header-1">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="image-container">
+                        <li><a href=""><img src="img/ruby.png" alt="ruby"></a></li>
+                        <li><a href=""><img src="img/css.png" alt="css"></a></li>
+                        <li><a href=""><img src="img/javascript.png" alt="javascript"></a></li>
+                        <li><a href=""><img src="img/html.png" alt="html"></a></li>
+                    </ul>
+                    <button class="btn" type="button"><span class="text">See project</span></button>
+                </div>
+            </div>
+            <div class="grid-item">
+                <div class="grid-item-top"></div>
+                <div class="grid-item-bottom">
+                    <h4 class="header-1">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="image-container">
+                        <li><a href=""><img src="img/ruby.png" alt="ruby"></a></li>
+                        <li><a href=""><img src="img/css.png" alt="css"></a></li>
+                        <li><a href=""><img src="img/javascript.png" alt="javascript"></a></li>
+                        <li><a href=""><img src="img/html.png" alt="html"></a></li>
+                    </ul>
+                    <button class="btn" type="button"><span class="text">See project</span></button>
+                </div>
+            </div>
+            <div>
+                <h1 class="header-2">About me</h1>
+                <img class="mask" src="img/Mask Group.png">
+
+                <p class="paragraph"> I can help you build a product , feature or website Look through some of my work
+                    and
+                    experience! If you like
+                    what you see and have a project you need coded,
+                    don’t hesitate to contact me.
+                </p>
+                <img class="frame" src="img/Frame.png">
```

Comment:
- [ ] If the image is not part of the page content, it should be added via background-image css property. That is what happens here too. Decorations are always backgrounded images, not img tags. Kindly add this background image with css background-image property.